plugins/eos/gs-plugin-eos.c

```diff
@@ -1341,6 +1347,22 @@ gs_plugin_eos_blacklist_app_for_remote_if_needed (GsPlugin *plugin,
 	return do_blacklist;
 }
 
+static gboolean
+gs_plugin_eos_remove_blacklist_from_usb_if_needed (GsPlugin *plugin,
+						  GsApp *app)
+{
+	if (gs_app_has_category (app, "Blacklisted") &&
+	    gs_app_has_category (app, "USB")) {
+		g_debug ("Removing blacklisting from '%s': app is from "
```

Comment:
Nitpick: capitalize "USB" in this message and join it with the string on the line above since I think that fits easily within the roughly 100+-line wrap of this code